encrypted-media-respec.html

```diff
@@ -2115,13 +2139,12 @@
                   <p>
                     If this object's <var>session type</var> is <a def-id="persistent-usage-record-session"></a>, store the <var>record of key usage</var>.
                   </p>
+                  <p class="note">
+                    This step may be run asynchronously, including after the document has unloaded.
```

Comment:
In order for `persistent-usage-record` sessions to work, this note would need to apply to the entire algorithm, which is problematic as I noted at line 2119 above.